index.html

```diff
@@ -102,10 +102,35 @@ <h3 class="proj-title">Multi-Post Stories Gain+Glory</h3>
             <li>JavaScript</li>
             <li>html</li>
           </ul>
-          <a class="proj-button" href="#">See Project</a>
+          <a id="button-project-1" class="proj-button" href="#">See Project</a>
         </div>
+        <!-- POPUP -->
+        <!-- <div id="project-1-popup" class="project-popup">
+          <div class="project-popup-container">
+            <img id="project-cross-1" src="./images/cancel-icon.svg" class="pop-project-cross" alt="cancel-icon">
+            <img class="popup-project-image" src="./images/project-image.svg">
+            <div class="popup-title-buttons-container">
+              <h3 class="project-popup-title">Keeping track of hundreds of components</h3>
+              <div class="popup-button-container upper-button-container">
+                <a class="proj-button" href="https://yothu.github.io/my-portfolio/">See Live<img src="./images/export-icon.svg"></a>
+                <a class="proj-button" href="https://github.com/Yothu/my-portfolio">See Source<img src="./images/github-icon-2.svg"></a>
+              </div>
+            </div>
+            <ul class="proj-languages proj-languages-j-start">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+            </ul>
+            <p class="project-popup-description">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+            <div class="popup-button-container bottom-button-container">
+              <a class="proj-button" href="https://yothu.github.io/my-portfolio/">See Live<img src="./images/export-icon.svg"></a>
+              <a class="proj-button" href="https://github.com/Yothu/my-portfolio">See Source<img src="./images/github-icon-2.svg"></a>
+            </div>
+          </div>
+        </div> -->
+        <!-- /POPUP -->
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Remove commented code from your project.